app/controllers/Admin.scala

```diff
@@ -250,7 +250,7 @@ class Admin(
       tagIds <- tagIdsFuture
       selectedSectionOption <- selectedSectionOptionFt
     } yield {
-      val capiKeyJson: Json = parser.parse(s"""{"CAPI_API_KEY": ${config.capiKey}}""").right.get
+      val capiKeyJson: Json = parser.parse(s"""{"CAPI_API_KEY": ${config.capiKey}}""").toOption.get
```

Comment:
@davidfurey this line seems to be throwing an error in CODE (I don't have Workflow admin permissions for PROD so can't check it there). It looks like this was the last commit to touch this code (though seems likely that the bug preceded this commit) -- do you have any context on what the code here's doing? Happy to try and raise a PR if it'd be useful.

Reply:
To reproduce:
https://workflow.code.dev-gutools.co.uk/admin/sections-and-tags

Logs: https://logs.gutools.co.uk/s/editorial-tools/app/discover#/doc/664d234d-de11-5f34-8ac5-9ee2089ac473/logstash-ed-tools-workflow-2024.11.21?id=cZ1JT5MBT0J6efU8a76d